charts/incubator/openbooks/Chart.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v2
+appVersion: 1.0.0
+description: openbooks helm package
+name: openbooks
+version: 2.0.1
```

```suggestion
appVersion: 2.0.1
description: openbooks helm package
name: openbooks
version: 1.0.0
```
Looks like you switcharoo'ed these versions. Also not sure if this should be in the incubator folder, if you feel it's "stable" we might as well move it to that folder :) Otherwise `version` should be `0.1.0`.